### ISSUE TITLE
✨ Move plugin errors entry to notification manager

### ISSUE THREAD

**Issue by goanpeca** [napari/napari#3100]
## 🚀 Feature

The plugins dialog could be simplified if the plugin errors were displayed as part of the notification manager?

<img width="210" alt="Screen Shot 2021-07-29 at 9 31 22 AM" src="https://user-images.githubusercontent.com/3627835/127510630-3d0132b6-1176-4103-ba93-7da6f2e656e7.png">

## Motivation

Simplify the UI, plugins menu and dialogs.

---

Pinging @liaprins-czi

**Comment by tlambert03** [comment]
agreed.  The main thing that we would either lose (or need to port) is the select box that let's you review/select all of the tracebacks collected by a specific plugin

(i.e. the notification manager still doesn't really have a "history")

**Comment by goanpeca** [comment]
>(i.e. the notification manager still doesn't really have a "history")

Indeed, we would need to do some work and maybe add "sections" to the notification manager so that plugins errors could be accessed on demand more easily.

>The main thing that we would either lose (or need to port)

Port sounds good, might need to think about what history looks like on the notification manager.

**Comment by liaprins-czi** [comment]
### Overview of design need
- **In a sentence or two, describe the problem to be solved for users**
Consider if / how to make plugin errors more consistent in experience with other (napari core) errors.

### What level of design is needed? (Choose all that apply)
_This section may be updated by the designer / UX researcher working on this issue_

- [ ] **User experience research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
- [X] **information flow / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
- [X] **Visual:** creating mockups, icons, etc
(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)

### Is design a blocker?
- [ ] **Yes:** engineering cannot proceed without a design first
- [X] **No:** engineering can create a first version, and design can come in later to iterate and refine

If selecting **Yes**, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?

### T-shirt size
- [ ] ●○○○○ XS: able to be done in a typical workday or two
- [ ] ●●○○○ S: able to do within one sprint, if also working on other things in the sprint
- [X] ●●●○○ M: able to do within one sprint, if it is the sole focus
- [ ] ●●●●○ L: will likely take more than one sprint to complete, but less than two
- [ ] ●●●●● XL: a very large experience to be determined or overhauled, will need to be broken down further
